cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/heartbeat/StreamWriterHeartbeat.java

```diff
@@ -26,30 +30,28 @@
 public class StreamWriterHeartbeat {
 
   /**
-   * Type of heartbeat, describing at what moment it is sent by a Stream writer.
+   * Type of information that describes one {@link StreamSize} object.
    */
-  public enum Type {
+  public enum StreamSizeType {
     /**
-     * Heartbeat sent during Stream writer initialization.
+     * Size sent during Stream writer initialization.
```

Comment:
Why need to differentiate betweeo init and regular? Isn't the heartbeat should always just carries the absolute size?